packages/run/README.md

```diff
@@ -0,0 +1,80 @@
+# @rollup/plugin-run
+
+🍣 A Rollup plugin which runs your bundles in Node once they're built.
+
+Using this plugin gives much faster results compared to what you would do with [nodemon](https://nodemon.io/).
+
+## Install
+
+Using npm:
+
+```console
+npm i -D @rollup/plugin-run
+```
+
+## Usage
+
+Create a `rollup.config.js` [configuration file](https://www.rollupjs.org/guide/en/#configuration-files) and import the plugin:
```

Comment:
`Usage` and `Options` sections need to be updated to match the format of the other plugins. This is one of the more difficult tasks of migrating a plugin.

Reply:
@shellscape  There seems to be some explanation about how the plugin itself works in the Readme. I did not want to remove that, and hence moved that to the Usage Section. We can remove that if you think its not needed.

Reply:
Please take a look at any of the other PRs that have been merged to migrate a plugin. They all follow the same, standard README layout. Information should not be removed, but it should be refactored to fit the standardized layout and structure.

Reply:
This will still need resolution before we can merge.

Reply:
@shellscape  I've tried to match the Readme's Usage and Options section for the most part. Is there something specific that you are looking for ? Maybe should we move the `how it works?` line to the top?

There aren't any options that it explicitly supports, but only pass-through options of `child_process.fork()`, so Ive mentioned it there. Do you want me to add the individual options?

Reply:
Looks good